js/script.js

```diff
@@ -1,87 +1,82 @@
+
+let bookData;
 
 class Book {
-  constructor(title = 'Nuevo libro', author = 'Shakespeare', id) {
+  constructor(title = 'New Book', author = 'John Doe', id) {
     this.title = title;
     this.author = author;
     this.id = id;
   }
 }
 
-// Create a function to add a new book to the collection
 function getBookInfo(title, author, id) {
-  const divAuthor = document.createElement('div');
-  const divTitle = document.createElement('div');
+  const div = document.createElement('div');
   const removeButton = document.createElement('button');
   const li = document.createElement('li');
 
-  // Add id and classes to the elements
-  divTitle.classList.add('author');
-  divAuthor.classList.add('title');
+  div.classList.add('book-info');
   removeButton.classList.add('remove');
   removeButton.setAttribute('id', `button${id}`);
-  removeButton.setAttribute('onclick', `javascript:removeListElement(${id})`);
+  removeButton.setAttribute('onclick', `javascript:StorageBooks.removeListElement(${id})`);
   li.classList.add('book');
   li.setAttribute('id', `book${id}`);
 
-  // Add content to elements
-  divTitle.innerHTML = title;
-  divAuthor.innerHTML = author;
+  div.innerHTML = `${title} by ${author}`;
   removeButton.innerText = 'Remove';
   removeButton.type = 'button';
 
-  // Append elements
-  li.appendChild(divTitle);
-  li.appendChild(divAuthor);
+  li.appendChild(div);
   li.appendChild(removeButton);
 
   return li;
 }
 
-// Storage data in the browser
-function storeData() {
-  localStorage.setItem('bookData', JSON.stringify(bookData));
-}
-
-// Load data in the browser
-function loadData() {
-  const data = localStorage.getItem('bookData');
-  if (data) {
-    bookData = JSON.parse(data);
-    bookData.forEach((book) => {
-      bookList.appendChild(getBookInfo(book.title, book.author, book.id));
-    });
+class StorageBooks {
+  static storeData() {
+    localStorage.setItem('bookData', JSON.stringify(bookData));
   }
-}
 
-loadData();
+  static loadData() {
+    bookData = [];
+    const data = localStorage.getItem('bookData');
+    if (data) {
+      bookData = JSON.parse(data);
+      bookData.forEach((book) => {
+        bookList.appendChild(getBookInfo(book.title, book.author, book.id));
+      });
+    }
+  }
 
-// Add element from list
-addBook.addEventListener('click', () => {
-  if (newTitle.value && newAuthor.value) {
-    // Create an id to the elements
-    const id = bookData[bookData.length - 1] ? bookData[bookData.length - 1].id + 1 : 1;
-    const book = new Book(newTitle.value, newAuthor.value, id);
-    bookData.push(book);
-    bookList.appendChild(getBookInfo(book.title, book.author, book.id));
-    storeData();
+  static addLiElement() {
+    if (newTitle.value && newAuthor.value) {
+      const id = bookData[bookData.length - 1] ? bookData[bookData.length - 1].id + 1 : 1;
+      const book = new Book(newTitle.value, newAuthor.value, id);
+      bookData.push(book);
```

Comment:
- [ ] [optional] It would be nice to use array methods that doesn't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of `push`